src/util.js

```diff
@@ -47,16 +51,45 @@ export function pathRankSort(a, b) {
 		bAttr = b.attributes || EMPTY;
 	if (aAttr.default) return 1;
 	if (bAttr.default) return -1;
-	let diff = rank(aAttr.path) - rank(bAttr.path);
-	return diff || (aAttr.path.length - bAttr.path.length);
+	let aRank = rank(aAttr.path),
+		bRank = rank(bAttr.path);
+	return (aRank < bRank) ? 1 :
```

Comment:
It won't work with subtraction with string ranks: I need `'5' < '543' < '55'`.

Using `<` and `>` will work for both numeric and string ranks - so gives that little extra flexibility if you need to try out / plug-in different ranking methods later.